RELEASING.md

```diff
@@ -87,4 +90,20 @@ for the selected Ubuntu distribution in all arches (supported architectures can
 
 ## Step 2: Copy packages to packages.osrfoundation.org
 
-TODO
+### Update reprepro importer config
+
+Create a PR against the `refactor_osrfbuild` branch in reprepro-updater repository and change the 
+[SourceVersion field to the new one released in the PPA](https://github.com/ros-infrastructure/reprepro-updater/blob/refactor_osrfbuild/config/packages.osrfoundation.org/openrobotics_dart_packages_from_ppa.yaml#L8).
+
+### Run reprepro-importer Jenkins test
+
+Rebuild [this build](https://build.osrfoundation.org/job/reprepro_importer/33) of reprepro-importer 
```

Comment:
Build 33 is no longer available. Should we just say use the latest job or create a new job?

Reply:
Mentioning the build was not a good idea, indeed. I just described the params here: dadb41e